Formula/s/streamrip.rb

```diff
@@ -159,7 +159,6 @@ class Streamrip < Formula
     url "https://files.pythonhosted.org/packages/c9/85/e24bf90972a30b0fcd16c73009add1d7d7cd9140c2498a68252028899e41/pycryptodomex-3.23.0.tar.gz"
     sha256 "71909758f010c82bc99b0abf4ea12012c98962fbf0583c2164f8b84533c2e4da"
   end
-
   resource "pygments" do
```

Comment:
we should add this into the pypi extra pkg list? and also smells like an upstream issue too

Reply:
i see, thanks!